src/Filtering/MathematicalMorphology/DilateUsingFunctionalGrayscale/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.16.3)
+
+project( DilateUsingFunctionalGrayscale )
+
+find_package( ITK REQUIRED
+  COMPONENTS
+  ITKCommon
+  ITKIOImageBase
+  ITKMathematicalMorphology
+  ITKIOPNG
+  )
+include( ${ITK_USE_FILE} )
+
+add_executable( DilateUsingFunctionalGrayscale Code.cxx )
+target_link_libraries( DilateUsingFunctionalGrayscale ${ITK_LIBRARIES} )
+
+install( TARGETS DilateUsingFunctionalGrayscale
+  DESTINATION bin/ITKSphinxExamples/Filtering/MathematicalMorphology
+  COMPONENT Runtime
+)
+
+install( FILES Code.cxx CMakeLists.txt
+  DESTINATION share/ITKSphinxExamples/Code/Filtering/MathematicalMorphology/DilateUsingFunctionalGrayscale
+  COMPONENT Code
+)
+
+enable_testing()
+add_test( NAME DilateAGrayscaleImageTest
+  COMMAND ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/DilateUsingFunctionalGrayscale
+    ${CMAKE_CURRENT_BINARY_DIR}/cthead1.png
+    Output.png
+    5
```

Comment:
It looks like this argument is unused?

Reply:
The parameter is now in use since we switched to `FlatStructuringElement` in place of `BinaryBallStructuringElement`.